schema/swagger.yml

```diff
@@ -87,60 +87,78 @@ paths:
 definitions:
   Fees:
     type: object
+    required:
+      - send
+      - vote
+      - secondSignature
+      - delegate
+      - multisignature
+      - dappRegistration
+      - dappWithdrawal
+      - dappDeposit
+      - data
     properties:
       send:
         type: string
-        example: 10000000
+        example: '10000000'
       vote:
         type: string
-        example: 100000000
+        example: '100000000'
       secondSignature:
         type: string
-        example: 500000000
+        example: '500000000'
       delegate:
         type: string
-        example: 2500000000
+        example: '2500000000'
       multisignature:
         type: string
-        example: 500000000
+        example: '500000000'
       dappRegistration:
         type: string
-        example: 2500000000
+        example: '2500000000'
       dappWithdrawal:
         type: string
-        example: 10000000
+        example: '10000000'
       dappDeposit:
         type: string
-        example: 10000000
+        example: '10000000'
+      data:
+        type: string
+        example: '10000000'
 
   # State Related Models
   NodeConstants:
     type: object
     required:
-    - epoch
-    - milestone
-    - nethash
-    - reward
-    - supply
-    - build
-    - commit
-    - version
-    - fees
+      - epoch
+      - milestone
+      - nethash
+      - reward
+      - nonce
+      - supply
+      - build
+      - commit
+      - version
+      - fees
     properties:
       epoch:
         type: string
+        format: date-time
         example: '2016-05-24T17:00:00.000Z'
       milestone:
-        type: string
-        example: 1081560729258
+        type: integer
+        example: 1
```

Comment:
Not a very good example - the lowest value is 100000000

Reply:
That was suggested by @Isabello https://github.com/LiskHQ/lisk/pull/933#discussion_r148745553

Reply:
Maybe we should look at how milestone is named, since its just decrementing the reward value... 

Please set to `500000000` as per the constants file as the first entry